build/Makefile

```diff
@@ -2,6 +2,9 @@
 POSSIBLE_TARGETS := module unit_tests run_unit_tests wheel sdist
 export POSSIBLE_TARGETS
 
+ADDITIONAL_POSSIBLE_TARGETS := test flake8
```

Comment:
POSSIBLE_TARGETS are all ran as part of `make all`. ADDITIONAL_POSSIBLE_TARGETS are targets you can do explicitly on the command line but are not ran as part of 'all'.